scripts/cfos/tests/README.md

```diff
@@ -0,0 +1,18 @@
+## Test schema file setup ##
+You can either:
+- replace the contents of 'scripts/cfos/dataset_tables_schema.json' file with the contents of the 'scripts/cfos/tests/test_schema.json' file temporarily OR
+- change the code in make_dataset_data_tables.py so that schema_json points to the test schema file instead of the real one
```

Comment:
can you make a "schema_file" optional input that defaults to the production one, rather than requiring the user to change the code or edit files?

Reply:
Instead, I added a "test" parameter that will run this using the test schema.